modules/json/src/json-converter.js

```diff
@@ -14,6 +14,9 @@ import {instantiateClass} from './helpers/instantiate-class';
 import parseJSON from './helpers/parse-json';
 
 const isObject = value => value && typeof value === 'object';
+const FUNCTION_IDENTIFIER = '@@=';
+const ENUM_IDENTIFIER = '@@#';
+const CONSTANT_IDENTIFIER = '@@!';
```

Comment:
What is constant vs. enum?

Reply:
Yes, I am not sure we need to identifiers / namespaces. 

If we add a required prefix to force evaluation of a string as a constant, both function-valued constants (like `MapController`, `LinearInterpolator`, ...) and integer/string valued enums (like `COORDINATE_SYSTEM.METER_OFFSETS`) should be able to go into the same string-to-value map in the JSON converter. 

The `JSONConverter` may need a little TLC in this area, the constant handling was done rather quickly.